src/cli/commands/run.js

```diff
@@ -121,12 +119,14 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
     if (pkgCommands.length) {
       reporter.info(`${reporter.lang('possibleCommands')}`);
       reporter.list('possibleCommands', pkgCommands, cmdHints);
-      await reporter
-        .question(reporter.lang('commandQuestion'))
-        .then(
-          answer => runCommand(answer.trim().split(' ')),
-          () => reporter.error(reporter.lang('commandNotSpecified')),
-        );
+      if (!flags.nonInteractive || !config.nonInteractive) {
```

Comment:
You should not need to check both `flags` and `config`. Also, I think you need 
```
if (!(flags.nonInteractive || config.nonInteractive))`
```

instead.

Reply:
@BYK You're right. Thanks for the feedback!